workspace.json

```diff
@@ -45,6 +45,7 @@
     "dh-shared-domain": "libs/dh/shared/domain",
     "dh-shared-environments": "libs/dh/shared/environments",
     "dh-shared-feature-flags": "libs/dh/shared/feature-flags",
+    "dh-shared-permission-guard": "libs/dh/shared/permission-guard",
```

Comment:
suggestion: consider renaming the library to `feature-authorization`. The "authorization" term encompasses better what the library is about.

Reply:
Fixed.